src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const avarageTime = 3;
+  const longTime = 7;
+  const costOfDay = 40;
+  const avarageTimeDiscount = 20;
```

Comment:
The variable name 'avarageTimeDiscount' has a typo. It should be 'averageTimeDiscount'. Also, it's better to name it more descriptive, for example 'shortTermDiscount'.